### ISSUE TITLE
Changes (2.1 vs 5)

### ISSUE THREAD

**Issue by BarneyHoundDog** [sungaila/PDFtoZPL#55]
### Question

First of all thanks so much for your hard work on this library - it's incredibly useful.

We have tried migrating our code from using 2.1 to 5. We are finding that in version 5 the ZPL generates a label that is much wider than the PDF. We are using the Conversion.ConvertPdf method and just passing in the pdf bytes. Can you provide any pointers on why the two versions give different results?

Thanks !

David

**Comment by sungaila** [comment]
Hi @BarneyHoundDog, I'm glad you like this library!

Which version did you migrate to? Is it `5.0.0` or `5.0.1`?

`5.0.0` had a bug where the default DPI was increased from `203` to `300` (this would make the label bigger than expected). In `5.0.1` this was fixed and reverted back.

If you're on the older version still, please update to the newest one and see if it fixes your problem.

**Comment by BarneyHoundDog** [comment]
> Hi @BarneyHoundDog, I'm glad you like this library!
> 
> Which version did you migrate to? Is it `5.0.0` or `5.0.1`?
> 
> `5.0.0` had a bug where the default DPI was increased from `203` to `300` (this would make the label bigger than expected). In `5.0.1` this was fixed and reverted back.
> 
> If you're on the older one, please update to the newest version and see if it fixes your problem.

Apologies for not being more exact: we were using 5.0.1.

**Comment by sungaila** [comment]
@BarneyHoundDog Thank you for bringing this to my attention. I've been testing this and noticed a issue were the PDF is rendered at 300 DPI still.

Until I fix this issue you can workaround this by setting 203 DPI yourself:
```csharp
PDFtoZPL.Conversion.ConvertPdfPage(yourPdfByteArray, pdfOptions: new(Dpi: 203));
```

Please note that the output width and height will differ by a few pixels. Since `2.1.0` the rounding of the output size has been changed.

**Comment by BarneyHoundDog** [comment]
Thanks for looking at this so promptly.

**Comment by sungaila** [comment]
@BarneyHoundDog I've released [PDFtoZPL 5.0.2](https://www.nuget.org/packages/PDFtoZPL/5.0.2) with a bugfix. You can now upgrade from 2.1 without changes or keep the mentioned workaround.

Please reopen this issue if the new version is not working for you.
